providers/amazon/src/airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -462,6 +497,24 @@ def execute_async(self, key: TaskInstanceKey, command: CommandType, queue=None,
         """Save the task to be executed in the next sync by inserting the commands into a queue."""
         if executor_config and ("name" in executor_config or "command" in executor_config):
             raise ValueError('Executor Config should never override "name" or "command"')
+        if len(command) == 1:
+            from airflow.executors.workloads import ExecuteTask
+
+            if isinstance(command[0], ExecuteTask):
+                workload = command[0]
+                ser_input = workload.model_dump_json()
+                command = [
+                    "python",
+                    "-m",
+                    "airflow.sdk.execution_time.execute_workload",
+                    "--json-string",
+                    ser_input,
+                ]
```

Comment:
I think this will show up in the logs? Are you ok with that?

Reply:
For now yes, there isn't really an easier way to get this into the containers without other external services. So as a basic implementation, we're okay with this.